static/vendor/node-snackbar/package.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Please remove most of the stuff from snackbar except for what we'll actually be using on the client.